__tests__/index.tests.js

```diff
@@ -39,3 +40,63 @@ describe('error 404', ()=>{
         })
     })
    
+    describe('GET /api', () => {
+      test('GET:200 sends an object with the endpoints to the client', () => {
+        return request(app)
+          .get('/api')
+          .expect(200)
+          .then((response) => {
+            expect(response.body.endpoints).toEqual(expect.any(Object));
+            expect(Object.keys(response.body.endpoints)).toEqual(
+              expect.arrayContaining(["GET /api", "GET /api/topics","GET /api/articles",  ]))
+              
+                     
+          });
+      });
+
+    }
+    )
+    describe('error 404 on GET /api bad path', ()=>{
+      test('GET:404', ()=>{
+          
+              return request(app)
+              .get('/apii')
+              .expect(404)
+     
+          })
+      })
+      describe('GET /api/articles/:article_id', () => {
+        test('GET:200 sends an object with the article corresponding to the id given to the client', () => {
+          
+          return request(app)
+            .get(`/api/articles/1`)
+            .expect(200)
+            .then((response) => {
+              console.log(response.body)
```

Comment:
We should remove console logs before pushing work. They are only intended to be used for debugging